packages/replayio/src/utils/recordings/printDeferredRecordingActions.ts

```diff
@@ -7,61 +7,69 @@ import { dim, statusFailed, statusPending, statusSuccess } from "../theme";
 import { formatRecording } from "./formatRecording";
 import { LocalRecording } from "./types";
 
+// log-update interferes with verbose output
+const isDebug = !!process.env.DEBUG;
+
 let logMessage: Function = logUpdate;
 let loggingDone: Function = logUpdate.done;
-if (process.env.DEBUG) {
-  // log-update interferes with verbose output
-  logMessage = () => {};
+if (isDebug) {
+  logMessage = console.log.bind(console);
   loggingDone = () => {};
 }
 
+function getRecordingStatus(recording: LocalRecording): string | undefined {
+  switch (recording.uploadStatus) {
+    case "failed":
+      return "(failed)";
+    case "uploading":
+      return "(uploading…)";
+    case "uploaded":
+      return "(uploaded)";
+  }
+}
```

Comment:
I think we should back out the changes to this utility.

The reason this I passed some of this stuff in as parameters was to support other async operations (like processing) which would print a different status suffix. I think we may need to re-add processing (I'm getting conflicting requests about this so far).

Reply:
I don't feel _too strongly_ about this either, but it does feel like we may be making more work for ourselves in the near future by hard-coding these labels.

Reply:
I don't feel _too strongly_ about this either, but it does feel like we may be making more work for ourselves in the near future by hard-coding these labels.

Reply:
I'm definitely up for moving this up if needed and letting the caller decide about it. I like the adjusted logic though as I put some time into figuring out today that some of the stuff I was expecting was swallowed completely because I used `DEBUG`. I think keeping it here will help to make sure that this is accounted for when we have to make those labels and statuses configurable for processing.

Reply:
Yeah to be clear, I wasn't complaining about the changes to the logic, only the removal of the parameters.

Reply:
I brought back those parameters but I'm using render props to handle this now. Please let me know what you think.